alinka/widget/containers/main_body/footer/application.py

```diff
@@ -31,7 +31,13 @@ def validate_document_data(self) -> None:
 
     def print_documents(self) -> None:
         self.validate_document_data()
-        generate_and_save_decision(form_data=self.document_data, generate=True)
+        destination_path = QFileDialog.getExistingDirectoryUrl()
+        if not destination_path.isValid():
+            return
```

Comment:
when that can be invalid? do we really want to NOT print here without any information to our user?